lib/extensions/vector2.dart

```diff
@@ -15,11 +15,12 @@ extension Vector2Extension on Vector2 {
   /// Creates a [Point] from the [Vector2]
   Point toPoint() => Point(x, y);
 
-  /// Creates a [Rect] starting from [x, y] and going the [Vector2]
-  Rect toRect(Vector2 to) => Rect.fromLTWH(x, y, to.x, to.y);
+  /// Creates a [Rect] starting from [x, y] and having the size of the
+  /// argument [Vector2]
+  Rect toPositionedRect(Vector2 size) => Rect.fromLTWH(x, y, size.x, size.y);
 
   /// Creates a [Rect] starting in origo and going the [Vector2]
-  Rect toOriginRect() => Rect.fromLTWH(0, 0, x, y);
+  Rect toRect() => Rect.fromLTWH(0, 0, x, y);
```

Comment:
I think `toPositionedRect` and `toRect` here are not 100% clear. I'm not sure what would be best, maybe `startRectWithSize` and `asSizeRect` ?

Reply:
I agree that they aren't that clear, I'm not convinced that your suggestions are more clear though.

Reply:
yeah I think we don't have the perfect one yet. I am ok with merging for now but let's keep thinking cc @renancaraujo @erickzanardo @wolfenrain if you have ideas

Reply:
It of course also depends on the motivation behind these methods. I can understand the `toPositionedRect`, tho I would indeed name it differently. But the `toRect` looks "useless" to me without context. 

So what are the usecases, and from there we can think of naming them, fitting with the context.

Reply:
`toRect` is used for example to convert the size vector to a `Rect` that the canvas can use.
Any ideas of what to name the `toPositionedRect`, and `toRect`?